cli/index.ts

```diff
@@ -61,7 +66,9 @@ async function main() {
     .option('--auth-param <name=value>', 'Additional variables to pass to the token endpoint', parseVariables, new Map())
     .action(capture('Import', ({ job }) => ({ job }), importCube))
 
-  return program.parseAsync(process.argv)
+  return trace.getTracer('cube-creator-cli').startActiveSpan('run', span => {
+    return program.parseAsync(process.argv).finally(span.end.bind(span))
```

Comment:
I'd suggest you add some error handling there to record the exception if there is one. You could also do that at other stages to make it clearer where the error comes from, but at least one at the root will help.

```suggestion
  return trace.getTracer('cube-creator-cli').startActiveSpan('run', async span => {
    try {
      return await program.parseAsync(process.argv)
    } catch (err) {
      span.recordException(err)
      span.setStatus({ code: SpanStatusCode.ERROR, message: err.message })
      throw err
    } finally {
      span.end()
    }
```